pkg/controller/elasticsearch/settings/merged_config.go

```diff
@@ -16,6 +17,8 @@ import (
 	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/volume"
 )
 
+var nodeAttrNodeName = fmt.Sprintf("%s.%s", esv1.NodeAttr, EnvNodeName)
```

Comment:
I believe typically these `node.attr`s are lowercase (just a nit). Also should we try to make it clear that `NODE` in this context does not mean the Elasticsearch node but the k8s node by maybe using a slightly different name?

Reply:
Do you think just `k8s_node_name`? I went this way to minimize the amount of mappings you have to keep in mind between ES attr name -> env var name -> downward api value. I can see that that might not be a big advantage over your proposed approach though

Reply:
I think I'd be +1 with `k8s_node_name`.

Reply:
Same